content/Watsonx/Assistant/104.md

```diff
@@ -32,33 +32,211 @@ Unless you thoroughly tested or experimented with your assistant as you complete
+
+Let’s walk through an example where an end user converses with your assistant. You will play the role of the builder or product manager, who reviews analytics data periodically to identify areas for improvement.
+
+First, engage in the following conversation with your assistant. Remember to have this conversation on your Lendyr website or on the Preview page, and not the Preview panel within the Actions page, as it will not register for analysis purposes.
+
+1. Hey I want to open a new account 4. How do I find my investor number?
+2. IRA 5. Speak to a live agent
+3. Investment I mean
+
+![](./images/104/image-016.jpg)
+
+Now, return to watsonx Assistant’s **Analyze (1)** page. Open the **Conversations (2)** tab and see this conversation appear in the logs (your last conversation will be at the very top). Click it **(3)** to see it displayed on the right **(4)**:
+
+![](./images/104/image-017.jpg)
+
+> Note that as watsonx Assistant’s AI learns from conversations, your interaction may look somewhat different than what is displayed above.
+
+Imagine you are the builder or product manager, and you identify a few areas for improvement:
+
+   • Individual Retirement Accounts (which most people would just abbreviate to IRAs) are a type of investment account. The assistant did not recognize this response as the end user’s attempt to select the investment account option.
+   • The assistant does not recognize the “How do I find my investor number?” or “Location of investor number” request.
+   • This conversation would have escalated to a live agent because the customer was not able to find the solution to their problem via the virtual assistant. However, note that you also do not have an action that recognizes the request to talk to an agent.
+
+You can iterate on your assistant to address these areas for improvement. For example, add **IRA** and **individual retirement account** as synonyms to the **Investment** option in the **Open a new account** action. You can begin immediately from the current view, by clicking on the action **(5)**:
+
+![](./images/104/image-018.jpg)
+
+This will open the **Open a new account** action. Click on **Step 1 (1)**. This is the step that asks whether the end user needs to open an investment account. Click **Edit response (2)**:
+
+![](./images/104/image-019.jpg)
+
+In the **Edit response** popup, click **Add synonyms** + (not shown), and then, as shown in the image below, add the appropriate synonyms to the **Investment option (1)**, then click **Apply (2)**:
+
+> **Note**: Pay attention that you are adding the synonyms to the Investment option and not one of the other options, as your order of options may differ from the image below.
+
+![](./images/104/image-021.jpg)
+
+Now, try using the new synonyms. You can do this without leaving the action build window:
+
+![](./images/104/image-022.jpg)
+
+Congratulations! You have successfully used watsonx Assistant’s analytics to improve an action in your assistant.
```

Comment:
You can customize the completion by stating the following. 

"Congratulations! You've reached the end of lab 104: Analytics. Click, [lab 105] to start the next lab.

Add any further instruction for this lab before congratulating for completion. Like 

Next, create a brand-new action called How do I find my investor number? Add location of investor number as one of the additional phrases. Invent your own brief answer to the question, either text or image. (You can easily find stock image URLs using Google Images.)

Reply:
updated